
robot_base_frame: X2/base_link
world_frame: world
#Navigation parameters
holonomic: false
angular_max_speed: 0.6  
linear_max_speed: 0.7 #0.6
angle_margin: 90 #for 1,5 rad/s and 0.6 m/s 10 degress works good in no holonomic way
dist_margin: 0.3
traj_timeout: 0.8 #0.025
a: 1.5 # for no holonomic 2 its okey with the 1.5 rad/s speed
b: 1.5
start_orientate_dist: 1.0

#security margin params
f: 2
inner_radius: 0.1
outer_radius: 0.15
full_laser_topic: /scanFromPC
security_stop_topic: /security_stop